tests/contrib/psycopg/test_psycopg.py

```diff
@@ -368,6 +368,24 @@ def test_user_specified_service(self):
         self.assertEqual(len(spans), 1)
         assert spans[0].service == "mysvc"
 
+    @TracerTestCase.run_in_subprocess(env_overrides=dict(DD_TRACE_SPAN_ATTRIBUTE_SCHEMA="v0"))
+    def test_span_name_v0_schema(self):
+        conn = self._get_conn()
+        conn.cursor().execute("""select 'blah'""")
+
+        spans = self.get_spans()
+        self.assertEqual(len(spans), 1)
+        assert spans[0].name == "postgres.query"
+
+    @TracerTestCase.run_in_subprocess(env_overrides=dict(DD_TRACE_SPAN_ATTRIBUTE_SCHEMA="v1"))
+    def test_span_name_v1_schema(self):
+        conn = self._get_conn()
+        conn.cursor().execute("""select 'blah'""")
+
+        spans = self.get_spans()
+        self.assertEqual(len(spans), 1)
+        assert spans[0].name == "postgres.query"
```

Comment:
also here `postgresql.query`

Reply:
I'm going to fix this as part of https://github.com/DataDog/dd-trace-py/pull/5632 which deals with service naming (this PR is dealing with operation name)

Reply:
as above, this is actually correct for this PR. Updating.